examples/dapperish.go

```diff
@@ -18,8 +18,8 @@ import (
 func client() {
 	reader := bufio.NewReader(os.Stdin)
 	for {
-		ctx, span := opentracing.BackgroundContextWithSpan(
-			opentracing.StartSpan("getInput"))
+		span := opentracing.StartSpan("getInput")
+		ctx := opentracing.BackgroundContextWithSpan(span)
```

Comment:
just FYI, a literally identical change just went in here: https://github.com/opentracing/basictracer-go/pull/8/files#diff-c36a63bd5139369ebe5b275a1fba1599R21